SupportClient/SupportClient/Controllers/FeedbackViewController.swift

```diff
@@ -0,0 +1,219 @@
+        let collectionViewImagePadding = (UIScreen.main.bounds.width - UIScreen.main.bounds.width / ErrorFileCollectionViewCell.imageResizingRatio) / 2.0
+        return UIEdgeInsets(top: 0, left: collectionViewImagePadding, bottom: 0, right: collectionViewImagePadding)
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumLineSpacingForSectionAt section: Int) -> CGFloat {
+        return 40
+    }
+
+}
+
+// MARK: - DZNEmptyDataSet
+extension FeedbackViewController: DZNEmptyDataSetSource, DZNEmptyDataSetDelegate {
+    
+    // If no screenshots/videos are added by the user, they are prompted to add files
+    func customView(forEmptyDataSet scrollView: UIScrollView!) -> UIView! {
+        
+        let customView = CustomUIView(width: view.layer.bounds.width, height: view.layer.bounds.height / 3)
+        customView.backgroundColor = UIColor.backgroundColor
+        
+        let noFileLabel = UILabel()
+        // Create ParagraphStyle to format label text
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+        paragraphStyle.lineSpacing = 10
+        // Tell the user there are no images/videos to display
+        noFileLabel.lineBreakMode = .byWordWrapping
+        noFileLabel.numberOfLines = 0
+        let title = "No Attached Files\n"
+        let subtitle = "Add screenshots or screen recordings from your photo library if they help to describe the bug"
+        let titleAttributes: [NSAttributedString.Key : Any] = [
+            NSAttributedString.Key.font: UIFont._17RobotoMedium!,
+            .paragraphStyle: paragraphStyle,
+            NSAttributedString.Key.foregroundColor: UIColor.titleColor
+        ]
+        let attributedText = NSMutableAttributedString(string: title, attributes: titleAttributes)
+        let subtitleAttributes = [
+            NSAttributedString.Key.font: UIFont._13RobotoRegular!,
+            NSAttributedString.Key.foregroundColor: UIColor.subtitleColor,
+            ]
+        attributedText.append(NSAttributedString(string: subtitle, attributes: subtitleAttributes))
+        noFileLabel.attributedText = attributedText
+        noFileLabel.sizeToFit()
+        
+        let addFileButton = UIButton()
+        addFileButton.setTitle("Add Image/Video", for: .normal)
+        addFileButton.setTitleColor(.gray, for: .normal)
+        addFileButton.titleLabel?.font = UIFont._17RobotoRegular
+        addFileButton.layer.cornerRadius = 5
+        addFileButton.layer.borderWidth = 1
+        addFileButton.layer.borderColor = UIColor.gray.cgColor
+        addFileButton.addTarget(self, action: #selector(handleAddFileButtonTap), for: .touchDragInside)
+        
+        customView.addSubview(noFileLabel)
+        customView.addSubview(addFileButton)
```

Comment:
can we move all of this code to be within the `CustomView` file? It'll help with potential reusability and just really cleans this file up